corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/query_spec.js

```diff
@@ -4,24 +4,30 @@ hqDefine("cloudcare/js/formplayer/spec/query_spec", function () {
     describe('Query', function () {
 
         describe('itemset', function () {
-            let QueryListView = hqImport("cloudcare/js/formplayer/menus/views/query"),
+
+            let keyQueryView;
+
+            before(function () {
+                const QueryListView = hqImport("cloudcare/js/formplayer/menus/views/query"),
                 Utils = hqImport("cloudcare/js/formplayer/utils/utils");
 
-            let QueryViewModel = Backbone.Model.extend(),
-                QueryViewCollection = Backbone.Collection.extend(),
-                keyModel = new QueryViewModel({
+                const QueryViewModel = Backbone.Model.extend();
+                const QueryViewCollection = Backbone.Collection.extend();
+                const keyModel = new QueryViewModel({
```

Comment:
I think you _can_ multiple assign `const` the way we were doing with `let` before, but this probably looks nicer for the sake of consistent indentation.